templates/member/admin/memberip.html

```diff
@@ -37,4 +37,24 @@
             {% endfor %}
         </ul>
     </div>
+
+    {% if ":" in ip %}
+    <p>
+        {% blocktrans %}
+            Liste des membres dont la dernière IP connue fait partie du bloc {{ network_ip }}
+        {% endblocktrans %}
+    </p>
+
+    <div class="members">
+        <ul>
+            {% for member in network_members %}
+                <li>{% include "misc/member_item.part.html" with member=member info=member.last_visit|format_date:True avatar=True %}</li>
+            {% endfor %}
+        </ul>
+    </div>
+
+    <p>
+        En IPV6, les adresses sont attribuées par bloc d'IP. Un bot de spam peut donc facilement changer d'adresse IP. Sont affichées ici tous les membres dont l'IPV6 fait partie du même bloc que l'IP demandée.
```

Comment:
Détails typographiques et explicatifs.

```suggestion
        En IPv6, les adresses sont attribuées par bloc d'IP. Un bot de spam peut donc facilement changer d'adresse IP au sein de ce bloc. Sont affichées ici tous les membres dont l'IPv6 fait partie du même bloc que l'IP demandée.
```